### ISSUE TITLE
refactor annotation processor (grpc) to create classes in new packages

### ISSUE THREAD

**Issue by hendrikebbers** [hashgraph/hedera-services#4236]
Based on https://github.com/hashgraph/hedera-services/issues/4235 we need to move a lot of classes to new packages. We need to do the same for the classes that are generated.

Configure the annotation processor to create the classes in the correct package

Part of epic https://github.com/hashgraph/hedera-services/issues/4239

**Comment by rbair23** [comment]
I think instead of going down this road, we should introduce our own gRPC handlers directly on top of http2. The [specification](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) is very small and easy to implement and conform to. Also, we do not want to use any of the generated grpc handling code that protoc generates, because we do not want to have the grpc handling code auto-parse the protobuf objects for us. Instead, we really do want to get the raw protobuf bytes because even though we need to parse them to do validation, we are going to pass the raw protobuf bytes down to the platform for gossip. So rather than getting pre-parsed protobuf objects, and then re-serializing them, we will have the raw bytes we can just pass down.

Plus, we want to use pbj for parsing and not what protoc generates. So there really is no value that I can see is using any of the grpc frameworks out there for us.

**Comment by rbair23** [comment]
Or maybe because `netty` is too low level, we would be better using `Helidon` as our web server (which is a layer on top of netty) and then either do our own grpc on top of that, or use the grpc that `Helidon` has. I used this approach in the POC, and I think it could work. Helidon is really nice to work with. My only complaint is that it is a huge library, and even huger still due to netty. I've implemented an HTTP/2 server, and it doesn't require a lot of code, but it is too much to take on at this time. So I think for now, we could use Helidon and their gRPC and switch out to something more lightweight in the future if we want to. Or maybe Helidon Nima will be good enough -- a nice web server with no netty dependencies.

Trimming down our set of dependencies is really important, but maybe not in the first go around.
